sdk/core/System.ClientModel/api/System.ClientModel.net6.0.cs

```diff
@@ -58,6 +58,15 @@ public enum ClientErrorBehaviors
         Default = 0,
         NoThrow = 1,
     }
+    public partial class ClientLoggingPolicy : System.ClientModel.Primitives.PipelinePolicy
+    {
+        public ClientLoggingPolicy(bool isLoggingEnabled = true, System.Collections.Generic.List<string>? loggedHeaderNames = null, System.Collections.Generic.List<string>? loggedQueryParameters = null, string? requestIdHeaderName = null, bool logContent = false, int maxLength = 4096, string? assemblyName = null) { }
+        public static System.ClientModel.Primitives.ClientLoggingPolicy Default { get { throw null; } }
+        public System.Collections.Generic.List<string> LoggedHeaderNames { get { throw null; } }
+        public System.Collections.Generic.List<string> LoggedQueryParameters { get { throw null; } }
```

Comment:
Would we want to represent everything we have in constructor parameters as properties on the policy?